extensions/zig_demo/module.zig

```diff
@@ -0,0 +1,249 @@
+const std = @import("std");
+const Allocator = std.mem.Allocator;
+const HashMap = std.HashMap;
+
+/// Core enumeration types.
+const Result = packed enum(i32) {
+    ok,
+    not_found,
+    bad_argument,
+    serialization_failure,
+    parse_failure,
+    bad_expression,
+    invalid_memory_access,
+    empty,
+    cas_mismatch,
+    result_mismatch,
+    internal_failure,
+    broken_connection,
+    unimplemented,
+};
+const LogLevel = packed enum(i32) { trace, debug, info, warn, err, critical };
+const FilterHeadersStatus = packed enum(i32) { ok, stop, continue_and_end_stream, stop_all_iteration_and_buffer, stop_all_iteration_and_watermark };
```

Comment:
Why `ok` instead of `continue`? I don't mind, but I'm curious about the resoning.

Reply:
It's a keyword.